app/client/src/sagas/EvaluationsSaga.ts

```diff
@@ -216,7 +216,12 @@ function* evaluateTreeSaga(
       isCreateFirstTree,
     );
 
-    yield fork(updateTernDefinitions, updatedDataTree, unEvalUpdates);
+    yield fork(
+      updateTernDefinitions,
+      updatedDataTree,
+      unEvalUpdates,
+      isCreateFirstTree,
+    );
```

Comment:
According to above suggestion.  
```suggestion
    yield fork(
      updateTernDefinitions,
      updatedDataTree,
      isCreateFirstTree,
      unEvalUpdates,
    );
```